_posts/2015-08-10-type-driven-development.html

```diff
@@ -44,7 +44,7 @@
     My problem was that I didn't know how long it took to handle a message, so the Polling Consumer would have to measure that as well, constantly updating its estimates of how long it takes to handle a message.
   </p>
   <p>
-    This wasn't a harder problem that I could <a href="http://blog.ploeh.dk/2014/09/25/faking-a-continuously-polling-consumer-with-scheduled-tasks">originally solve it in a highly coupled imperative fashion</a>. I wasn't happy with that implementation, though, so I'm happy that there's a much more <em>incremental</em> approach to the problem.
+    This wasn't a sufficiently hard problem to thwart my <a href="http://blog.ploeh.dk/2014/09/25/faking-a-continuously-polling-consumer-with-scheduled-tasks">originally solving it in a highly coupled imperative fashion</a>. I wasn't happy with that implementation, though, so I'm happy that there's a much more <em>incremental</em> approach to the problem.
```

Comment:
Doesn't the change to a gerund transform the sentence from active voice into a passive(r) voice?